giraffe/README.md

```diff
@@ -70,19 +70,14 @@ A React-based visualization library powering the data visualizations in [InfluxD
 
 ## Example Using Flux [](#example-using-flux)
 
-When generating the table through a Flux result:
-
-- call the `fromFlux` utility function on the csv that is generated by Flux
-- get the table in the returned object from calling `fromFlux`
-
-Here is an example of turning a result in comma separate values (CSV) from Flux into a table and rendering it without optional properties:
+Use the string containing the comma separate values (CSV) from Flux as the `fluxResponse` property:
```

Comment:
we should keep both usages in the documentation. since parsing large CSVs is a relatively expensive operation, we still need to be able to parse flux closer to the injection point (by the API) so that we can memoize it. i see this PR as making a better, fuller, API.. not a migration strategy towards a contract change

Reply:
Ok i will put the other usage back into the docs.